rust/kcl-lib/src/std/extrude.rs

```diff
@@ -96,11 +99,16 @@ pub async fn extrude(exec_state: &mut ExecState, args: Args) -> Result<KclValue,
     args = {
         sketches = { docs = "Which sketch or sketches should be extruded"},
         length = { docs = "How far to extrude the given sketches"},
+        tag_start = { docs = "A named tag for the face at the start of the extrusion, ie. the original sketch" },
+        tag_end = { docs = "A named tag for the face at the end of the extrusion, ie. the opposite face to the original sketch" },
```

Comment:
s/opposite face to original sketch/the new face created by extruding the original sketch

wdyt

Reply:
nice